src/frontends/tensorflow_lite/tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
+-c ../../../bindings/python/constraints.txt
 numpy>=1.16.6,<1.25.0
```

Comment:
why it still remained with versions and does not use constaints

Reply:
Because `numpy` versions differ between different `requirements.txt` files under the scope of the `constraints.txt`. This is sadly a limitation of pip `-c` flag, there are more details in the PR description.

Reply:
I believe we can align it everywhere, now that `onnx==1.13.1` has been merged into master. However, this PR already does like five different things and we need to merge it soon. I would prefer doing the `numpy` alignment in a separate PR in order to save time.

Reply:
@p-wysocki, please create a ticket to fix it this release.
@mlukasze, fyi.

Reply:
I'm aware, but thanks for mentioning anyway :)

Reply:
@rkazants Done in https://github.com/openvinotoolkit/openvino/pull/16652, let's wait for green CI with review.